src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -169,8 +177,11 @@ private void LaunchDynamo(bool isCheckboxChecked)
         /// <summary>
         /// Once main window is initialized, Dynamic Splash screen should finish loading
         /// </summary>
-        private void FinishLoadingDynamicScreen()
+        private void OnStaticScreenReady()
         {
+            // Stop the timer in any case
+            loadingTimer.Stop();
+            loadingTimer = null;
```

Comment:
@filipeotero Stop the timer regardless `if` true or not below to make sure timer is always stopped